app/helpers/report_formatter_helper.rb

```diff
@@ -28,6 +28,7 @@ def review_response_map(params, _session = nil)
     # If review response is required call review_response_report method in review_response_map model
     @reviewers = ReviewResponseMap.review_response_report(@id, @assignment, @type, @review_user)
     @review_scores = @assignment.compute_reviews_hash
+    @author_feedback_scores = @assignment.compute_author_feedback_scores
```

Comment:
this line is causing failures in Travis. Need to supply method with appropriate parameters.